src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -312,6 +320,21 @@ typedef struct {
     int tag_bits;
 } MPIDI_OFI_capabilities_t;
 
+typedef struct {
+    fi_addr_t dest;
+    int rank;
+    int state;
+} MPIDI_OFI_conn_t;
+
+typedef struct MPIDI_OFI_conn_manager_t {
+    int mmaped_size;
+    int max_n_conn;
+    int n_conn;
+    int next_conn_id;
+    int *free_conn_id;
+    MPIDI_OFI_conn_t *conn_list;
```

Comment:
`/* The list of connection structs to track the outstanding dynamic process connections. */`